deployment/inception.sh

```diff
@@ -69,10 +69,20 @@ echo ........ Creating App Insights
 # Create OMS Workspace
 echo ........ Creating OMS Workspace
 K8_DEPLOYMENT_DIAGSA_NAME="${PROJECT_NAME}diagsa"
+
+# Generate log analytics parameters file
+LOG_ANALYTICS_OMS_PARAMS=$(<logAnalyticsOms.parameters.json)
+LOG_ANALYTICS_OMS_PARAMS=$(jq --arg workspaceName $PROJECT_NAME-$AZURE_LOCATION-ws '.parameters.workspaceName.value=$workspaceName' <<< "$LOG_ANALYTICS_OMS_PARAMS")
+LOG_ANALYTICS_OMS_PARAMS=$(jq --arg storageAccountName $K8_DEPLOYMENT_DIAGSA_NAME '.parameters.applicationDiagnosticsStorageAccountName.value=$storageAccountName' <<< "$LOG_ANALYTICS_OMS_PARAMS")
+LOG_ANALYTICS_OMS_PARAMS=$(jq --arg resourceGroup $COMMON_RESOURCE_GROUP '.parameters.applicationDiagnosticsStorageAccountResourceGroup.value=$resourceGroup' <<< "$LOG_ANALYTICS_OMS_PARAMS")
+echo $LOG_ANALYTICS_OMS_PARAMS > logAnalyticsOms.parameters.temp.json
+
 az storage account delete --name $K8_DEPLOYMENT_DIAGSA_NAME --resource-group $COMMON_RESOURCE_GROUP --yes
-az storage account create --name $K8_DEPLOYMENT_DIAGSA_NAME --resource-group $COMMON_RESOURCE_GROUP --location eastus --sku Standard_LRS
```

Comment:
I believe we hardcoded the value in the script because not all locations support OMS work spaces.

Reply:
Good catch. Updated here: https://github.com/Microsoft/containerized-microservices-pipeline/pull/75/commits/2924bd60f09bae2e99a606b14ae1f24f224ce007